src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/Constants.cs

```diff
@@ -63,7 +63,7 @@ public static class Constants
         /// <summary>
         /// The default policy API version.
         /// </summary>
-        public static readonly string PolicyApiVersion = "2016-04-01";
+        public static readonly string PolicyApiVersion = "2016-12-01";
```

Comment:
Per the resource schema seen on Azure Portal, the latest version for policy is 2016-12-01:
```json
        {
            "resourceType": "policyDefinitions",
            "locations": [],
            "apiVersions": [
                "2016-12-01",
                "2016-04-01",
                "2015-10-01-preview"
            ]
        },
        {
            "resourceType": "policyAssignments",
            "locations": [],
            "apiVersions": [
                "2016-12-01",
                "2016-04-01",
                "2015-10-01-preview"
            ]
        },
```